rfcs/NNNN-lit-3.0.md

```diff
@@ -0,0 +1,125 @@
+---
+Status: Active
+Champions: @justinfagnani
+PR: https://github.com/lit/rfcs/pull/16
+---
+
+# Lit 3.0
+
+A proposal for the next major release of Lit
+
+## Objective
+
+Allow us to make a few breaking changes that will let us reduce some engineering complexity, and very lightly reduce bundle size.
+
+### Goals
+- Require no code changes from developers using non-deprecated Lit 2.0 APIs
+- Drop IE support
+- Publish ES2021
+- Small cleanups (remove Lit pre-2.0 compatibility workarounds)
```

Comment:
What about improving some types? Not a behavior change, and not a major type change, but one that might break a very small number of users.

Someone prototyped a fix for https://github.com/lit/lit/issues/3709 internally and a very small number of targets were broken, seems like a good candidate, and something we could land

Reply:
We included this: https://github.com/lit/lit/pull/3710